playbooks/robusta_playbooks/util.py

```diff
@@ -0,0 +1,20 @@
+
+import re
+from typing import Dict, List
+
+REGEXP_FIND_SELECTOR = r'\{\{\s*labels\.([^\s\}]+)\s*\}\}'
+
+def resolve_selectors(labels:Dict, selectors:List[str]) -> List[str]:
```

Comment:
I would add this above
`build_selector_query` in 
`/Users/avirobusta/git/robusta/src/robusta/integrations/kubernetes/custom_models.py`
and import from there

Reply:
look at `format_pod_templated_string`, you might be able to use it in a similar manner

Reply:
Done